permissions.md

```diff
@@ -16,7 +23,8 @@ If you wish to get going quickly these commands will be good to know!<br/>
 `_perms group <name> link <role>` - This is how you can link a group to a specific Discord role. More info [here](#group-linking)<br/>
 
 ## Groups
```

Comment:
You should probably add move stuff here. also specify that it's order from top to bottom stating with 1

Reply:
Btw I'm refering to the groups section. github doesn't really make that clear